USSF-ORBIT/ussf-portal-client
# feat: Drag and drop support for links within a collection

# SC-154



## Proposed changes


- A user is is able to re-arrange links (canonical and custom) within a collection via drag and drop



## Setup


To test:
- Run `yarn services:up` in your local directory
- Run ussf-portal-cms locally with `yarn dev`
- Pull down this branch and run locally with `yarn dev`
- Login and while viewing My Space, verify that you can see the icon to the left of each link in a collection.
- Click and hold the icon while dragging the link to a new position within the collection. Verify that the newly moved link remains where you dropped it.
- Refresh the page and verify that the new order of the links persists.
---

## Code review steps

### As the original developer, I have

- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):

### As code reviewer(s), I have

- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):

### As a designer reviewer, I have

- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
